### PR TITLE
Update to pyproject.toml with setup.cfg

### PR DESCRIPTION
**Pull request by AisenGinn**
Replaced setup.py with pyproject.toml and setup.cfg.
Work as following:
1. pip install -q build
2. python -m build --wheel
3. pip install -e '.[_extras_req_]'

### CODE REVIEW

**Comment by Gamenot** [comment]
Hey, this is good work. I think the follow-up left is the following:

- Update `make format` command
- Address the deprecation warnings

**Comment by Gamenot** [comment]
Oh, on second thought, your additional commit user does not exist. You will either have to associate that email with your account or use git to fix the history of commits in this change-set.

**Comment by AisenGinn** [comment]
> Oh, on second thought, your additional commit user does not exist and is causing issues for the auto-commit. You will either have to associate that email with your account or use git to fix the history of commits in this change-set.

OK I will do it

**Review by Gamenot — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    License :: OSI Approved :: MIT License
+
+[options]
+packages = find:
+include_package_data = True
+zip_safe = True
+python_requires = >=3.7
+install_requires =
```
Comment (Adaickalavan) [inline setup.cfg:22]: Ensure the requirements are updated with the latest requirements from the develop branch prior to merging.

**Review by Adaickalavan — APPROVED** [review]

**Pull request merged**